app-text/texlive_texmf/additional-files/createSubpackageInfos.pl

```diff
@@ -0,0 +1,95 @@
+#!/bin/perl
+
```

Comment:
Should I add a license header here?

As to which one, I suppose it would have to be GPL v2 because it depends on such code. But I am open to MIT too, if that is possible.